content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,127 @@
+deprecated or are removed in a Kubernetes version. The API versions found out of
+support should be updated to the supported version and a new version of the chart
+released. The API version is defined by the `kind` and `apiVersion` fields. For
+example, here is a removed `Deployment` object API version in Kubernetes 1.16:
+
+```yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+```
+
+## Helm Users
+
+You should audit the charts that you use (similar to the [chart mantainer](#chart-maintainers))
+and identify any charts where API verions are deprecated or removed in a
+Kubernetes version. For the charts identified, you need to check for the latest
+version of the chart (which has supported API versions) or update the chart yourself.
+
+Additionally, you also need to audit any charts deployed (i.e. Helm releases) checking
+again for any deprecated or removed API versions. This can be done by getting
+details of a release using the `helm get` command.
+
+The means for updating a Helm release to supported APIs depends on your findings
+as follows:
+
+1. If you find deprecated API versions only then:
+  - Perform a `helm upgrade` with a version of the chart with supported Kubernetes
+  API versions
+  - Add a description in the upgrade, something along the lines to not perform a
+  rollback to a Helm version prior to this current version
+2.  If you find any API version(s) that is/are removed in a Kubernetes version then:
+  - If you are on a Kubernetes version where the removed API version(s) are in a
+  later Kubernetes version (i.e. you are on Kubernetes 1.15 but the removed API
+  versions are in Kubernetes 1.16):
+    - Follow the step 1 procedure
+  - Otherwise:
+    - You need to edit the release manifest in-cluster updating the API
+    versions to supported APIs. See [Updating API Versions of a Release Manifest](#updating-api-versions-of-a-release-manifest) for more details
+
+> Note: In all cases of updating a Helm release with supported APIs, you should
+never rollback the release to a version prior to the release version with the
+supported APIs.
+
+If you don't update a release as suggested previously, you will have an error
+similar to the following when trying to upgrade a release in a Kubernetes version
+where its API version(s) is/are removed:
+
+```
+Error: UPGRADE FAILED: unable to build kubernetes objects from current release
+manifest: unable to recognize "": no matches for kind "Deployment" in version
+"apps/v1beta1"
+```
+
+Helm fails in this scenario because it attempts to create a diff patch between
+the current deployed release (which contains the Kubernetes APIs that are removed
+in this Kubernetes version) against the chart you are passing with the
+updated/supported API versions. The underlying reason for failure is that when
+Kubernetes removes an API version, the Kubernetes Go client library can no longer
+parse the deprecated objects and Helm therefore fails calling the library. Helm
```

Comment:
Missing "when" in "fails when calling the library"?

Reply:
Sure, fixed.